app/src/main/java/amazingme/database/FirebaseHelper.java

```diff
@@ -74,4 +62,62 @@ public static void signOut() {
         getFirebaseAuthInstance().signOut();
     }
 
+    private static void onCompleteRegistrationHandler(final Activity activity, final Task<AuthResult> task) {
+        final ActivityManager activityManager = ActivityManager.getInstance();
```

Comment:
@masteryoda123 what are you used to with singletons? 

i might have you change this line to ActivityManager.getInstance.goTo() on line 74, just to emphasize it's a singleton. 

but persuade me otherwise. haha it's just coding style and i'm curious as to why you did it the way you did. maybe it'll be better.

Reply:
it's just style, really. I'll change it

Reply:
haha no no no. just wondering. :) 

i'm down for anything I just thought one was more clear.